functions/python-chatbot/serverless.yml

```diff
@@ -0,0 +1,22 @@
+service: chatbot-python-function
+
+provider:
+  name: scaleway
+  runtime: python310
+
+plugins:
+  - serverless-scaleway-functions
+
+package:
+  patterns:
+    - '!node_modules/**'
+    - '!bin/**'
+    - '!.gitignore'
+    - '!.idea'
+    - '!.git/**'
+
+functions:
+  chatbot-python:
+    handler: app/chatbot.handle
+    env:
+      database_model: english-corpus.sqlite3
```

Comment:
add a note in readme to explain where `english-corpus.sqlite3` comes from and why needed